style.css

```diff
@@ -16,51 +16,89 @@ body {
 }
 
 /* hero section */
+
 #hero {
   display: flex;
   flex-direction: column;
   justify-content: center;
   align-items: center;
 }
 
+#hero header {
+  position: relative;
+  width: 100vw;
+}
+
 /* header design */
 
 #navbar {
-  background-color: white;
-  width: 100vw;
-  height: 7.2vh;
+  display: block;
+  width: 0;
   position: absolute;
   top: 0;
-  left: 0;
-  display: flex;
-  align-items: center;
-  justify-content: end;
+  right: 0;
+  height: 92vh;
+  background-color: #fff;
+  border-radius: 0 0 20px 20px;
+  z-index: 3;
+  transition: 0.5s;
 }
```

Comment:
[OPTIONAL]
- [ ] You can consider adding `position: fixed; height: 100vh; overflow: hidden` to the navbar class to prevent the mobile menu from scrolling with the page.